src/stampit.js

```diff
@@ -46,7 +46,16 @@
     return dst;
   }
 
-  var assign = _Object.assign || _mergeOrAssign.bind(0, assignOne);
+ // var assign = _Object.assign || _mergeOrAssign.bind(0, assignOne);
+ var assign = (ob, ...o) => {
+  o.forEach((obj) => {
+    if (typeof obj !== 'undefined') {
+      Object.defineProperties(ob, Object.getOwnPropertyDescriptors(obj));
```

Comment:
I believe all we need to do is to insert this line after  the line 44. And it will just work I think. Try that.

Reply:
i try this, but stil can not pass test:
```js
function _mergeOrAssign(action, dst) {
    return slice.call(arguments, 2).reduce(action, dst);
  }

  function assignOne(dst, src) {
    if (src) {
      // var keys = objectKeys(src);
      // for (var i = 0; i < keys[_length]; i++) {
      //   dst[keys[i]] = src[keys[i]];
			// }
			Object.defineProperties(dst, Object.getOwnPropertyDescriptors(src));

    }
    return dst;
  }

 // var assign = _Object.assign || _mergeOrAssign.bind(0, assignOne);
 var assign = (ob, ...o) => {
  o.forEach((obj) => {
    if (typeof obj !== 'undefined') {
      Object.defineProperties(ob, Object.getOwnPropertyDescriptors(obj));
    }
  });

  return ob;
};

  function isFunction(obj) {
    return typeof obj == 'function';
  }
```

Reply:
i got it, thank you very much:)